docs/configuration.md

```diff
@@ -1752,6 +1752,13 @@ showDF(properties, numRows = 200, truncate = FALSE)
     Executable for executing R scripts in client modes for driver. Ignored in cluster modes.
   </td>
 </tr>
+<tr>
+  <td><code>spark.r.shell.command</code></td>
+  <td>R</td>
+  <td>
+    Executable for executing R shell.
```

Comment:
maybe "Executable for executing the SparkR shell. Ignored in cluster modes.". Don't have strong opinion on  the wording, just be sure to be consistent with the existing description for other 2 related options. The old descriptions can be updated together.